activerecord/lib/active_record/relation.rb

```diff
@@ -291,6 +291,11 @@ def empty?
     end
 
     # Returns true if there are no records.
+    #
+    # When a pattern argument is given, this method checks whether elements in
+    # the Enumerable match the pattern via the case-equality operator, `===`.
```

Comment:
Some nitpicks about the punctuation.
I'm not sure a comma is correct here, but I could be wrong (not a native English speaker).

A colon doesn't look right either.
```suggestion
    # the Enumerable match the pattern via the case-equality operator: `===`.
```

Maybe:
```suggestion
    # the Enumerable match the pattern via the case-equality operator (`===`).
```

Reply:
+1 for the parentheses version

Reply:
Yes. It makes. I updated with the parentheses version.